ddtrace/internal/writer/writer.py

```diff
@@ -576,25 +560,21 @@ def _downgrade(self, payload, response, client):
             # These endpoints share the same encoding, so we can try sending the
             # same payload over the downgraded endpoint.
             return payload
-        raise ValueError()
 
-    def _send_payload(self, payload, count, client):
-        # type: (...) -> Response
+    def _send_payload(self, payload, count, client) -> Response:
```

Comment:
Type hint? I _think_ it should also be:
```suggestion
    def _send_payload(self, payload: bytes, count: int, client: WriterClientBase) -> Response:
```